Makefile

```diff
@@ -5,22 +5,38 @@ render/precincts-2163.shp: out/nation.gpkg
 	ogr2ogr -t_srs EPSG:2163 -overwrite -skipfailures $@ $<
 
 out/nation.gpkg: \
-        out/18-indiana/state.gpkg out/20-kansas/state.gpkg \
```

Comment:
Splitting these onto separate lines to minimize future merge conflicts when multiple PRs are in-flight.